### DISCUSSION TITLE
[Feature Request] Windows 10 old search icon

### DISCUSSION THREAD

**Discussion by Apis035** [valinet/ExplorerPatcher#1719] [Ideas]
The search button on Windows 10 looks too thick and felt out of place.
An option to revert this to the old icon would be nice.

Current:
![image](https://github.com/valinet/ExplorerPatcher/assets/53115851/fcdc00b6-4cef-4422-b66a-5184a2210cd1)

Old:
![image](https://github.com/valinet/ExplorerPatcher/assets/53115851/265242cb-840a-4078-a828-f1735f236e21)

**Comment by milestprower92** [comment]
> The search button on Windows 10 looks too thick and felt out of place. An option to revert this to the old icon would be nice.
> 
> Current: ![image](https://private-user-images.githubusercontent.com/53115851/257018848-fcdc00b6-4cef-4422-b66a-5184a2210cd1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzMTg1OTYsIm5iZiI6MTcwMjMxODI5NiwicGF0aCI6Ii81MzExNTg1MS8yNTcwMTg4NDgtZmNkYzAwYjYtNGNlZi00NDIyLWI2NmEtNTE4NGEyMjEwY2QxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjExVDE4MTEzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhNDI4ZTZmZGNkMTQ2NjNmYjQyMzkxNDEwYWRjY2EzOTllYThhZmY3OWZlMjA2MWE2Y2I5MzkyNjIzY2NiYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.egd1CT7ikJP0YdfKhX4vz8u0-LBshzgoYW5jHGju0ZE)
> 
> Old: ![image](https://private-user-images.githubusercontent.com/53115851/257018854-265242cb-840a-4078-a828-f1735f236e21.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzMTg1OTYsIm5iZiI6MTcwMjMxODI5NiwicGF0aCI6Ii81MzExNTg1MS8yNTcwMTg4NTQtMjY1MjQyY2ItODQwYS00MDc4LWE4MjgtZjE3MzVmMjM2ZTIxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjExVDE4MTEzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFkYmViYjg2YjViY2Q5NmJmNjMwYTAxZmM4MGVjZmJiZTViYjkyNDhlMmY0NDVlMWFmM2Y1NjJjNzUxNjU0NWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lGNUTb8Yd_Vxq8RbL9q57uNc5JfcIIAYrEpVhg59jGI)

the only method to restore the old search icon in windows 10 is by disabling this feature ID: 41868508

**Reply by alphabetlorefan2003** [reply]
F this, the creator must add an option or else.

**Reply by milestprower92** [reply]
> F this, the creator must add an option or else.

Disable this feature ID on ViveTool:
41868508

**Reply by yoelwep13578** [reply]
> Disable this feature ID on ViveTool:
> 41868508

I CD-ed my CMD into ViVe Tool folder and I ran command with `vivetool /disable /id:41868508`. This is correct?

**Comment by WellDonyEddy** [comment]
Sadly it stopped working after today's update KB5052077

**Reply by pyrates999** [reply]
That's a preview update: https://support.microsoft.com/en-us/topic/february-25-2025-kb5052077-os-build-19045-5555-preview-4ec11d48-1649-4549-846d-b3197a7becda

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

**Comment by Kichura** [comment]
This is now intentionally done when using the reimplemented Windows 10 taskbar.
